cypress/e2e/repo.cy.js

```diff
@@ -1,57 +1,119 @@
 describe('Repo', () => {
   beforeEach(() => {
-    cy.login('admin', 'admin');
-
     cy.visit('/dashboard/repo');
 
     // prevent failures on 404 request and uncaught promises
     cy.on('uncaught:exception', () => false);
   });
 
-  describe('Code button for repo row', () => {
-    it('Opens tooltip with correct content and can copy', () => {
-      const cloneURL = 'http://localhost:8000/finos/git-proxy.git';
-      const tooltipQuery = 'div[role="tooltip"]';
+  describe('Anonymous users', () => {
+    it('Prevents anonymous users from adding repos', () => {
+      cy.get('[data-testid="repo-list-view"]')
```

Comment:
It might be a good idea to alias some of these long element identifiers to make tests more readable. Not exactly a problem for smaller tests, but definitely good if the frontend component or tests get refactored:

```js
cy.get('[data-testid="repo-list-view"]').as('repoList');
cy.get('@repoList').find('[data-testid="add-repo-button"]'); // chainable
```

Reply:
Added issue for this #1143.